### PR TITLE
Add some docs

### PR DESCRIPTION
**Pull request by ehashman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,69 @@
...
+   ```
+
+   to fetch dependencies.
+
+3. Obtain a fernet key. You only need to generate this once; you can store it
+   in your `.lein-env` later, per step 2.
+
+   ```clojure
+   lein repl
+   cloudpassage-poller.core=> (require 'fernet.core)
```
Comment (derwolfe) [inline README.md]: s/poller/lib

File: README.md [hunk 2]
```diff
@@ -1,14 +1,69 @@
-# cloudpassage-lib
+# cloudpassage-lib #
```
Comment (derwolfe) [inline README.md]: Should there be a trailing #?

File: doc/cloudpassage-api.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+3. When making a call to the API, pass the access token in the Authorization
+   header field. Specify the authentication scheme as "Bearer", followed by a
+   space and then the token. For example:
+
+   ```
+   Authorization: Bearer ffad76cc550110fc4c84a18397b6e104
+   ```
+
+## Events ##
```
Comment (derwolfe) [inline doc/cloudpassage-api.md]: Since we aren't exposing anything specific to events, I think this could be removed.

File: README.md (hunk 2 above)
Comment (ehashman) [inline README.md]: It's a style thing, I prefer it that way because it's symmetric and I'm silly :) The trailing # doesn't render.

Comment (ehashman) [inline README.md]: Made the other fixes but didn't see this early enough to include it. Do you want me to go back and make this change as well?

Comment (derwolfe) [inline README.md]: Yes, please. :-)

**Pull request merged**